src/pykeen/pipeline/api.py

```diff
@@ -524,8 +524,8 @@ def replicate_pipeline_from_config(
 
 def _iterate_moved(pipeline_results: Iterable[PipelineResult]):
     for pipeline_result in pipeline_results:
-        pipeline_result.model.device = resolve_device("cpu")
-        pipeline_result.model.to_device_()
+        pipeline_result.model.to(resolve_device("cpu"))  # FIXME make in-place
```

Comment:
https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.to

it is already in-place